doc/source/ref/2d-decompositions-overview.rst

```diff
@@ -0,0 +1,55 @@
+
+Overview of multilevel wavelet decompositions
+=============================================
+
+There are a number of different ways a decomposition can be performed for
+multiresolution analysis of n-dimensional data.  Here we will review the three
+approaches currently implemented in PyWavelets.  2D cases are illustrated, but
+each of the approaches extends to the n-dimensional case in a straightforward
+manner.
+
+
+Standard Multilevel Discrete Wavelet Transform
+----------------------------------------------
+
+The most common approach to the multilevel discrete wavelet transform involves
+further decomposition of only the approximation subband at each subsequent
+level.  This is also sometimes referred to as the Mallat decomposition
+[TODO: REF].  In 2D the discrete wavelet transform, produces four sets of
+coefficients corresponding to the 4 possible compinations of the wavelet
+decomposition filters over the two separate axes.  (In n-dimensions, there
+are 2**n sets of coefficients).  This approach is illustrated below.
+
+In PyWavelets it is implemented for n-dimensional data by `wavedecn` and the
+inverse by `waverecn`.  1D and 2D versions of these routines also exist.
+
+.. image:: ../_static/Mallat.png
```

Comment:
again would be good to commit the scripts rather than the pngs. do you still have them?

Reply:
I made these in inkscape, but at some later point I did make a Matlab function that could create these type of plots.  I will look into using that instead

Reply:
I made these in inkscape, but at some later point I did make a Matlab function that could create these type of plots.  I will look into using that instead